### PR TITLE
Console input2 fx3

### PR DESCRIPTION
**Pull request by ik1xpv**
Tentative debug console input and output via USB from / to ExtIO debug console in Windows compiled in Debug.
In Interface.h #define _DEBUG_USB_ enables USB debug and disable serial trace

### CODE REVIEW

**Comment by ik1xpv** [comment]
Here a debug trace with RX888mk2 via USB. ExtIO is compiled in debug
DebugFX3USB2
To enter debug command move focua to ExtIO console and type  Enter or ?Enter to have the command list.
threads lists the active threads... etc.

**Comment by ik1xpv** [comment]
Note when using hardware serial port the FX3 numbers every second 1,  2,  3; ...
This is disabled when debug via USB to reduce the display mess.
You can manually read DMA counting  using  ?Enter.

File: Core/RadioHandler.cpp [hunk 1]
```diff
@@ -344,8 +344,8 @@ void RadioHandlerClass::CaculateStats()
 	BytesXferred = 0;
 	SamplesXIF = 0;
 
-	uint16_t maxlen = 64;
-	uint8_t  debdata[64];
+	uint16_t maxlen = 100;
```
Comment (howard0su) [inline Core/RadioHandler.cpp:347]: suggest a constant value in Interface.h

File: SDDC_FX3/Application.h [hunk 2]
```diff
@@ -26,7 +26,10 @@
 #include "cyu3externcstart.h"
 #include "i2cmodule.h"
 
-#define MAXLEN 64		// max buffer debug len
+#include "../Interface.h"
+
+#define TRACESERIAL		/* enable the trace to serial port*/
+#define MAXLEN (100)		// max buffer debug len
```
Comment (howard0su) [inline SDDC_FX3/Application.h:32]: as previous comment, move the constant shared between firmware and PC software into interface.h

File: SDDC_FX3/Application.h [hunk 3]
```diff
@@ -65,14 +68,17 @@
...
+
+#ifndef _DEBUG_USB_  // #include "../Interface.h"
+#define DebugPrint (CyU3PDebugPrint)
+#else
+#define DebugPrint (DebugPrint2USB)
+#endif
 
 // detect pin for HF103
-#define LED_KIT		(54) // This is also UART_CTS
+#define LED_KIT		   (54)		// This is also UART_CTS
```
Comment (howard0su) [inline SDDC_FX3/Application.h:81]: unrelated change

**Review by howard0su — APPROVED** [review]

File: Core/RadioHandler.cpp (hunk 1 above)
Comment (ik1xpv) [reply Core/RadioHandler.cpp]: I placed define MAXLEN_D_USB (100)  in Interface.h  Thanks !

File: SDDC_FX3/Application.h (hunk 2 above)
Comment (ik1xpv) [reply SDDC_FX3/Application.h]: Thanks!

File: SDDC_FX3/Application.h (hunk 3 above)
Comment (ik1xpv) [reply SDDC_FX3/Application.h:80]: Here may be  I got problems with the tab in different editors. 
I want to change the way in witch BBRF103 is detected and not to use LED_KIT anymore...
